en/flight_modes/altitude.md

```diff
@@ -1,68 +0,0 @@
-# Altitude Flight Mode

```

Comment:
FYI only, split this into a FW and an MC version. That does result in more docs, but they are easier to read. I think it makes sense when the behaviour and parameters are actually quite different.